helm/app/values-prod.yaml

```diff
@@ -11,23 +11,18 @@ resources:
       cpu: 500m
       memory: '1Gi'
 
-pgBouncer:
-  requests:
-    cpu: 250m
-    memory: 16Mi
-  limits:
-    cpu: 250m
-    memory: 32Mi
+proxy:
+  pgBouncer:
+    requests:
+      cpu: 250m
```

Comment:
suggest cpu request be 100m or less

Reply:
Sorry I missed this message. Did you think it should be `1m` like we have for the rest in `values.yaml`? Also the app requests just above this are the same as the limits as well at `500m`, should I update that?

Reply:
Should be fine at `1m - 10m` and yes, we should update that one as well if you haven't already. The intention is that the pod is "burstable" and we are being good community members by just requesting ~average the usage and setting the limit to allow it to burst during "high" traffic or expensive transactions.

Reply:
Sounds good. Yeah that is what we have been doing though I can't remember why, Alec and I changed the prod values to increase the requests hoping it would solve some issue though I will revert it.